glossary.asciidoc

```diff
@@ -39,6 +39,9 @@ Block::
 Blockchain::
 	A list of blocks validated by the proof-of-work system, each linking to its predecessor all the way to the genesis block. This varies from the Bitcoin protocol's in that it does not have a block limit, it instead uses varying gas limits.
 
+Byzantium::
+  Byzantium is the first of two hard forks for Metropolis development stage.
```

Comment:
Perhaps, here we could link it with the Ice Age HF following the **"EIP-649**: Byzantium Difficulty Bomb Delay", which proposes to delay the difficulty bomb for approximately one and a half year.